apps/financial-aid/web-veita/src/components/TableBody/TableBody.tsx

```diff
@@ -21,13 +21,20 @@ import {
   calcDifferenceInDate,
   getTagByState,
 } from '@island.is/financial-aid-web/veita/src/utils/formHelper'
+import { useApplicationState } from '../../utils/useApplicationState'
```

Comment:
@island.is/financial-aid-web/veita/src/utils/useApplicationState, maybe this instead?